torch/_inductor/scheduler.py

```diff
@@ -1477,7 +1477,15 @@ def __init__(self, nodes: List[ir.Operation]) -> None:
         self.create_foreach_nodes()
         self.topological_sort_schedule()
         self.logged_slow_fusion: Set[Tuple[str, str]] = set()
+        self.nodes = spmd_bucketing.bucketing_per_blcok(self.nodes)
         self.fuse_nodes()
+        # do prefetching reordering
+        if self.post_grad_graph_id == 0:
```

Comment:
Does id=0 means forward graph and id=1 means backward graph?
If so please comment out in the code to make it more readable.